backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardwrapper/BpkCardWrapper.kt

```diff
@@ -41,6 +42,7 @@ fun BpkCardWrapper(
     headerContent: @Composable () -> Unit,
     cardContent: @Composable () -> Unit,
     modifier: Modifier = Modifier,
+    cardPadding: BpkCardPadding = BpkCardPadding.Small,
```

Comment:
this is the actual change - before we always assumed that the card would have the small padding, but that's not always the case